app/javascript/packs/fieldCharacterLimits.js

```diff
@@ -0,0 +1,27 @@
+const user_profile_field_ids = [
```

Comment:
(comment): We should probably use camelcased variable names in JS files.

Reply:
🙈 Being a multilingual programmer can be hazardous!

Reply:
LOL tell me about it, I use `binding.pry` in my JS code sometimes 😂